ui/app/components/oidc/client-form.js

```diff
@@ -62,10 +62,14 @@ export default class OidcClientForm extends Component {
           // note: when editing the UI removes any additional assignments previously configured via CLI
           this.args.model.assignments = ['allow_all'];
         }
+        // if TTL components are toggled off, set to default lease duration
+        const { idTokenTtl, accessTokenTtl } = this.args.model;
+        // value returned from API is a number, and string when from form action
+        if (Number(idTokenTtl) === 0) this.args.model.idTokenTtl = '24h';
```

Comment:
Not something to address here (by any means) but I wonder if this is something we could address in the TTL component down the line. I'm assuming this conversion happens in lots of place that use this component. Just thinking out loud here.

Reply:
The component doesn't return a string, it's the attribute on the model changes depending on whether it comes from the backend (so when editing) or if we interact with it in the form. 

I believe it has to be a string when we interact with the form because of how we join/split and adjust the TTL based on the unit (s, h, m or d)